beefy-gadget/src/lib.rs

```diff
@@ -231,14 +231,21 @@ where
 	}
 
 	fn handle_finality_notification(&mut self, notification: FinalityNotification<Block>) {
-		info!(target: "beefy", "Finality notification: {:?}", notification);
+		debug!(target: "beefy", "🥩 Finality notification: {:?}", notification);
 
 		if self.should_vote_on(*notification.header.number()) {
+			let commitment = Commitment {
+				payload: notification.header.hash(),
```

Comment:
This should be configurable, we want this to be a MMR root. I'd put a TODO in all places like that, but it's fine :)